src/main/scala/reson/server/Server.scala

```diff
@@ -1,53 +1,64 @@
 package reson
 
-import com.twitter.finagle.exp.mysql.{Error, ServerError}
-import com.twitter.finagle.http.Method._
-import com.twitter.finagle.http._
-import com.twitter.finagle.http.Status._
-import com.twitter.finagle.http.path._
-import com.twitter.finagle.{SimpleFilter, Http, Service}
+import com.twitter.finagle.http.Method.{Get, Options}
+import com.twitter.finagle.http.Status.{Conflict, InternalServerError, NotFound}
+import com.twitter.finagle.http.path.{/, Path, Root}
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.finagle.mysql.ServerError
+import com.twitter.finagle.{Http, Service, SimpleFilter}
 import com.twitter.util.{Await, Future}
+import org.json4s.JsonAST.JValue
+import org.json4s.JsonDSL._
 import reson.db.MySQL
-import reson.server.{RequestParser}
-import util.Transformers._
-import rapture.json._
-import rapture.json.jsonBackends.jackson._
+import reson.server.RequestParser
+import reson.util.Transformers._
 
 /**
-  * Created by sscarduzio on 23/12/2015.
-  */
+ * Created by sscarduzio on 23/12/2015.
+ */
 object Server extends App {
-  def handle(req: Request, table: String, f: String => Future[String]): Future[Response] = {
+  def handle(req: Request, table: String,
+    f: String => Future[JValue]): Future[Response] = {
     for {
       query <- RequestParser.parse(req, table).toFuture
-      qString <- query.materialize.toFuture
-      dbResp <- f(qString)
+        qString <- query.materialize.toFuture
+        dbResp <- f(qString)
     } yield mkResp(dbResp)
   }
 
   lazy val route = Service.mk[Request, Response] { req =>
     (req.method, Path(req.path)) match {
       case (Get, Root) => MySQL.tableList.map(mkResp)
-      case (Get | Options, Root / (table: String)) => handle(req, table, MySQL.read)
+      case (Get | Options, Root / (table: String)) => handle(req, table,
+        MySQL.read)
```

Comment:
are you using a very narrow line length?

Reply:
I'm using std. 80 chars line length... I can revert this change if you like.

Reply:
yes please set it at least 2x (160 chars) for Scala! thanks

Reply:
Ok + reverted line wraps to be on par with original code.

Reply:
Thank you @tzeman77! Good job :)